samples/glean/src/main/java/org/mozilla/samples/glean/MainActivity.kt

```diff
@@ -97,7 +97,9 @@ open class MainActivity : AppCompatActivity(), ExperimentUpdateReceiver.Experime
         textViewExperimentStatus.setBackgroundColor(Color.WHITE)
         textViewExperimentStatus.text = getString(R.string.experiment_not_active)
 
-        activeExperiments = Nimbus.getActiveExperiments()
+        val nimbus = GleanApplication.nimbus
+
+        activeExperiments = nimbus.getActiveExperiments()
```

Comment:
```suggestion
        GleanApplication.nimbus.getActiveExperiments()
```

Any reason why it can't be like this?

Reply:
The `nimbus` object is used in several places in this example.